### ISSUE TITLE
Are we allowed to use the urdf files of the variants of the 8 training scenes in which the locations of objects are randomized?

### ISSUE THREAD

**Issue by naokiyokoyama** [StanfordVL/iGibsonChallenge2021#12]
Are we allowed to use the urdf files of the variants of the 8 training scenes in which the locations of objects are randomized?

**Comment by ChengshuLi** [comment]
Yes, you are allowed to do so. The pedestrians in Social Nav and the addtionally added small objects in Interactive Nav already have randomized locations. I guess you are referring to the furniture objects in the scene?

**Comment by Gengshucheng** [comment]
hi, I watched your presentation on Youtube about the igibson chanllenge, and I'm trying to learning something about it, could you share  the source code of the last year? thanks a lot！
